cli/cli/kurtosis_config/config_version/config_version.go

```diff
@@ -8,6 +8,7 @@ const (
 	ConfigVersion_v0 ConfigVersion = iota
 	ConfigVersion_v1
 	ConfigVersion_v2 // Fixed a typo in Kubernetes config, `enclave-size-in-Megabytes` -> `enclave-size-in-megabytes`
-	ConfigVersion_v3
-	ConfigVersion_v4
+	ConfigVersion_v3 // adds logs aggregator config
+	ConfigVersion_v4 // adds engine-node-name to KubernetesClusterConfig
+	ConfigVersion_v5 // adds GraflokiConfig to KurtosisClusterConfig
```

Comment:
Grafloki seems fine for naming variables in the code, but for user-facing stuff I'd explicitly call it GrafanaLokiConfig (no reason to shorthand where clarity is much more important than speed). Will wait to see if the actual config object below calls it this

Reply:
renaming to `GrafanaLokiConfig`